### DISCUSSION TITLE
Filling several records in a row

### DISCUSSION THREAD

**Discussion by MagicDidou** [filamentphp/filament#13087] [Help]
### Package

Form builder

### Package Version

v3.x

### How can we help you?

Hi everybody,

I would like to know if any of you know a way (plugin, component, other..) to allow the user to enter several records successively without having a blank form at each record (don' know if I'm clear...).

Imagine user opening the Form which displays a grid (kind of excel sheet) with columns headers and user fill several rows, each of them being one new record in the database. The blank form at each record is a bit heavy and for high number of records to enter.

Any ideas ?
Thx

**Comment by leandrocfe** [comment]
Why not a importAction?
https://filamentphp.com/docs/3.x/actions/prebuilt-actions/import#overview

**Reply by MagicDidou** [reply]
I have Import/Export Actions but the user does not want to move back & forth from Excel and the Filament App.
I thought I can manage this using a RelationManager resource and repeater...

**Reply by leandrocfe** [reply]
Maybe you can create a [custom page](https://filamentphp.com/docs/3.x/panels/pages#creating-a-page), add a [form](https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component#adding-the-form) and use a [table repeater](https://filamentphp.com/plugins/awcodes-table-repeater).

**Reply by MagicDidou** [reply]
I think this is what I will do as I cannot see a way to do this using "standard resources".

Thanks

**Reply by leandrocfe** [reply]
Yes, I think this would be a custom form
